packages/commercetools/api-client/__tests__/api/customerUpdateMe.spec.ts

```diff
@@ -1,6 +1,6 @@
 import customerUpdateMe from '../../src/api/customerUpdateMe';
 
-jest.mock('../../src/api/customerUpdateMe/defaultMutation.ts', () => '');
+jest.unmock('../../src/api/customerUpdateMe/defaultMutation.ts');
```

Comment:
Can you explain why there is an unmock?

Reply:
@Baroshem I think it's redundant. Deleted.